File: src/main/java/help/commands/store/InfoCommands.java
package help.commands.store;

import help.commands.dto.commands.HelpCommandType;
import help.commands.dto.commands.HelpCommands;
import help.commands.dto.response.CommandsResponse;
import help.commands.dto.response.CommandsResponseItem;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Component;

import java.util.List;

@Component
public class InfoCommands {

    public ResponseEntity<CommandsResponse> getInfo(){
        return commands();
    }

    private ResponseEntity<CommandsResponse> commands(){
        return ResponseEntity.ok().body(new CommandsResponse(
                List.of(
                        new CommandsResponseItem(
                HelpCommandType.INFO,
                new HelpCommands(
                        "You are in the help service! Those commands will allow you to watch list of commands",
                        "user -commands <- will show you commands for your account"
                )
                )
                )
        ));
    }
}

File: src/main/java/help/commands/store/InfoCommands.kt
package help.commands.store

import help.commands.HelpCommandType
import help.commands.HelpCommands
import org.springframework.stereotype.Component

@Component
class InfoCommands {
    private val commands: Map<HelpCommandType, HelpCommands> = mapOf(
        HelpCommandType.INFO to HelpCommands(
            description = "You are in the help service! Those commands will allow you to watch list of commands",
            command = "/--user-commands"
        )
    )

    fun getInfo() = commands
}